entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+lint=""
+warnings=""
+notes=""
+
+while [ "$#" -gt 0 ]; do
+    arg=$1
+    case $1 in
+        -l|--lint) lint="--lint"; shift;;
+        -w|--deny-warnings) warnings="--deny-warnings"; shift;;
+        -n|--deny-notes) notes="--deny-notes"; shift;;
+        *) break;;
+    esac
+done
+
+exclusions=$1
```

Comment:
@a-frantz - OK, as much as I hate parsing args in bash, this should do it. Are there any other Sprocket arguments at the moment that I should support. I didn't see anything else.

Reply:
No I think this should do it!